### ISSUE TITLE
Icon doesn't match system aesthetic of new macOS

### ISSUE THREAD

**Issue by yannikbloscheck** [ImageOptim/ImageOptim#325]
The current icon is great, but it doesn't match the new system aesthetic for icons of macOS Big Sur.
I actually liked the previous icon style better, but now that the rounded rectangles are here it just looks weird, if an icon falls out of line.
I created a temporary replacement icon just for me (see below). So I know it's really difficult to get the idea of compressing just in a square - even Apple's Compressor icon isn't that great, but I hope an official new icon is already in planing.
<img src="https://user-images.githubusercontent.com/1571006/99843189-6faf7280-2b71-11eb-9834-ecc059981337.png" alt="Icon" width="128" height="128"/>

**Comment by kornelski** [comment]
Thanks for the suggestion.

I also weep for the loss of good macOS style and loss distinctive icon shapes. 

The metallic rounded frame around the icon is the way to go. However, rest of this icon is IMHO too dark. It could be a top-down view of a trashcan :)

Could you try giving it some front metal plate? Here's a quick mockup:

![Untitled](https://user-images.githubusercontent.com/72159/99861302-94b1de80-2b8d-11eb-85ee-9e15c405eb44.png)

**Comment by yannikbloscheck** [comment]
I like the idea with the three screws inside. I tried only one screw before and that looked weird. I modified my icon accordingly:

<img src="https://user-images.githubusercontent.com/1571006/99877479-ca42e000-2bfe-11eb-949b-721d11e42215.png" alt="Icon" width="64" height="64"/>

If you want the original Sketch, PDF etc. files, just let me know.

**Comment by NiklasBr** [comment]
> I also weep for the loss of good macOS style and loss distinctive icon shapes.

++

The frame makes it look like an oven, and I can't unsee it 😄 

https://images-na.ssl-images-amazon.com/images/I/81TUyKlpm1L._AC_SL1500_.jpg

But, it does look nice.

**Comment by kornelski** [comment]
Yes, please upload the Sketch file (dropping ZIP in this github issue should be enough)

**Comment by yannikbloscheck** [comment]
Here you go:
[Icon.zip](https://github.com/ImageOptim/ImageOptim/files/5579593/Icon.zip)

**Comment by kornelski** [comment]
Two more contenders:

https://twitter.com/caspianievers/status/1330786605242597376

<img height=128 src="https://user-images.githubusercontent.com/72159/99956525-110b1400-2d7e-11eb-9ca0-9dfe3bb64e12.jpg">

https://twitter.com/emoryzanef/status/1330792699448836097

<img height=128 src="https://user-images.githubusercontent.com/72159/99956652-4879c080-2d7e-11eb-96f5-1499329bbbfe.jpg">

**Comment by yannikbloscheck** [comment]
I like the general style of the first and it looks beautiful in big, but it has a few problems, if I actually put it in the dock at a smaller size.
Then the screws are too big and close to another and the nice texture on them gets lost and unrecognisable. Same problem with the lightning in the bottom part. In the dock just a weird pink line remains visible.

The second one just doesn't work for me, because of the handle. It would be good to have the handle somewhere, but anywhere you put it the handle just looks too thin or small in my experience. In this case the handle is also a bit off perspective to make it more recognisable. A good idea in theory and Apple does the same with the guitar in the new GarageBand logo, but it just really weirds me out every time I see it 😄

**Comment by yannikbloscheck** [comment]
The thing with the handle left me no peace. This is the best I came up with while honouring the perspective:
<img src="https://user-images.githubusercontent.com/1571006/100095193-69174880-2e5a-11eb-8823-ec621ced4081.png" alt="Icon" width="128" />

**Comment by kornelski** [comment]
Previous icons for reference: 

![icon-change](https://user-images.githubusercontent.com/72159/100107174-476d8f80-2e61-11eb-8cf9-34f88bb9f971.png)

**Comment by kornelski** [comment]
I'm afraid the icon style doesn't leave room for the hand-operated handle. It's time to replace it with an engine :)

**Comment by yannikbloscheck** [comment]
> ![icon-change](https://user-images.githubusercontent.com/72159/100107174-476d8f80-2e61-11eb-8cf9-34f88bb9f971.png)

I'm familiar with the second one, but I have actually never seen the first one. It looks a bit like a death trap 😄


> I'm afraid the icon style doesn't leave room for the hand-operated handle. It's time to replace it with an engine :)

I have transferred the few small lightning improvements form the handle version to the engine version:
<img src="https://user-images.githubusercontent.com/1571006/100115006-4f81fb00-2e72-11eb-8730-5181e021855a.png" alt="Icon" width="128" />
[Icon.zip](https://github.com/ImageOptim/ImageOptim/files/5590905/Icon.zip)

**Comment by boesbo** [comment]
Sheets come off the frame, this is against the Big Sir style

**Comment by benjibee** [comment]
I really like the vise, I'd hate to see it go! I think it communicates what the app does really well, and for me hearkens back to Stuffit Expander and the like, which I think is pretty cool. Also looking at examples of the new icon style from Apple I think all of the contenders here are a bit on the dark side, no?

Looking around online I found [a website offering user-contributed alternate icons](https://macosicons.com/) for macOS and this little diddy, which isn't so far off from some of these designs:

![image](https://user-images.githubusercontent.com/620119/101335836-ff476780-3879-11eb-82d0-00dfde07f89e.png)

Apple's human user interface guidelines (HUIG) [give plenty of examples](https://developer.apple.com/design/human-interface-guidelines/macos/overview/whats-new-in-macos/) where a tool or implement breaks the borders of the square for added effect:

![image](https://user-images.githubusercontent.com/620119/101335968-2867f800-387a-11eb-811a-f71937fadcf8.png)

They even explain when this is reasonable:

_Although the design language strongly encourages visual consistency, it doesn’t preclude judicious expressiveness. For example, the Preview, Xcode, and TextEdit icons continue to combine depictions of the physical objects that best convey the app’s core purpose, while incorporating the new shape, perspective, and shadow._

Just my 2¢ as a user who's had this icon in their dock since it looked like it was meant to crush humans 😬

**Comment by Zabriskije** [comment]
In case you guys are still interested in changing the icon, I've made two:

<img width="256" src="https://user-images.githubusercontent.com/101254295/161423579-d0e748a5-9508-417e-b427-c47e9304f3d1.png"> <img width="256" src="https://user-images.githubusercontent.com/101254295/161422917-e809dbe8-7181-4859-865c-5ea0972560d5.png">

They're based on Octaviotti design, and the background used is found as header of the ImageOptim site.
The one without background pattern is more in line with Apple HIG.

[Link to the .icns](https://github.com/Zabriskije/MacOS-Icons/tree/main/ImageOptim)

**Comment by mrleblanc101** [comment]
I like the simplicity of this one from [Tom Dai](https://macosicons.com/#/u/Tom%20Dai) on macosicons
<img width="512" alt="icon" src="https://github.com/ImageOptim/ImageOptim/assets/17392251/4c9f21cf-fce1-471a-9f01-18d11f3c2b9e">

**Comment by mrleblanc101** [comment]
Here we are 5 years later, and Apple changed the icon shape again.
Hope to see the app updated to the new HIG.
